crates/languages/src/rust/runnables.scm

```diff
@@ -1,6 +1,11 @@
 (
-    (attribute_item (attribute) @attribute
-        (#match? @attribute ".*test"))
+    (attribute_item (attribute
+        [((identifier) @_attribute)
+        (scoped_identifier (identifier) @_attribute)
+            ])
+        (#eq? @_attribute "test"))
```

Comment:
Will this work with `gpui::test`, `tokio::test` and others?

Reply:
Yep, all of the above :P